### PR TITLE
DFPL-328: Placement - Local Authority "Notice of hearing"

### PR DESCRIPTION
**Pull request by DanCatchpole**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-328

### Change description ###
- Remove 'notice' page from LA placement flow
- Allow for the LA solicitor to choose which respondents should receive the notice of hearing (once created by the admin - this creation flow + associated notifications will be in DFPL-330)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseField/CareSupervision/eps-managing.json [hunk 1]
```diff
@@ -566,6 +566,13 @@
     "UserRole": "[EPSMANAGING]",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
```
Comment (jamesrferguson1) [inline ccd-definition/AuthorisationCaseField/CareSupervision/eps-managing.json:571]: Reomve this

File: ccd-definition/AuthorisationCaseField/CareSupervision/judiciary.json [hunk 2]
```diff
@@ -3422,20 +3422,41 @@
     "UserRole": "caseworker-publiclaw-judiciary",
     "CRUD": "R"
   },
+  {
```
Comment (jamesrferguson1) [inline ccd-definition/AuthorisationCaseField/CareSupervision/judiciary.json:3425]: Add access to the label

File: ccd-definition/AuthorisationCaseField/CareSupervision/superuser.json [hunk 3]
```diff
@@ -286,6 +286,13 @@
     "UserRole": "caseworker-publiclaw-superuser",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
```
Comment (jamesrferguson1) [inline ccd-definition/AuthorisationCaseField/CareSupervision/superuser.json:290]: Add access to the lable

File: ccd-definition/CaseField/CareSupervision/manageOrders/sectionHeaders.json [hunk 4]
```diff
@@ -125,6 +125,15 @@
     "SecurityClassification": "PUBLIC",
     "Searchable": "N"
```
Comment (jamesrferguson1) [inline ccd-definition/CaseField/CareSupervision/manageOrders/sectionHeaders.json:126]: Move sectionHeaders out of manage orders. (maybe to caseField)

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java [hunk 5]
```diff
@@ -78,29 +89,25 @@ public AboutToStartOrSubmitCallbackResponse handleDocumentsUploaded(@RequestBody
...
-
-        final PlacementEventData eventData = placementService.preparePayment(caseData);
+        PlacementEventData eventData = placementService.prepareNotification(caseData);
+        putFields(caseProperties, eventData, PLACEMENT_GROUP, NOTICE_GROUP);
 
-        caseProperties.putIfNotEmpty("placement", eventData.getPlacement());
-        caseProperties.putIfNotEmpty("placementPaymentRequired", eventData.getPlacementPaymentRequired());
-        caseProperties.putIfNotEmpty("placementFee", eventData.getPlacementFee());
+        eventData = placementService.preparePayment(caseData);
+        caseDetails.getData().put("placement", eventData.getPlacement());
```
Comment (jamesrferguson1) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java:106]: Should be caseProperties

File: service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java [hunk 6]
```diff
@@ -451,13 +451,16 @@ public ApplicationsBundle getApplicationBundleByUUID(UUID elementId) {
...
     private final Selector childSelector;
     private final Selector othersSelector;
+    private final Selector respondentsSelector;
     private final Selector personSelector;
     private final Selector careOrderSelector;
     private final Selector newHearingSelector;
     private final Selector appointedGuardianSelector;
 
     private final String orderAppliesToAllChildren;
     private final String sendOrderToAllOthers;
```
Comment (jamesrferguson1) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java:461]: Do we need these?

File: service/src/main/java/uk/gov/hmcts/reform/fpl/model/PlacementNoticeDocument.java [hunk 7]
```diff
@@ -25,6 +25,11 @@
     private String recipientName;
     private UUID respondentId;
 
+
```
Comment (jamesrferguson1) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/model/PlacementNoticeDocument.java:28]: Do we need this as we're removing the Notice of placement stuff?

**Review by jamesrferguson1 — CHANGES_REQUESTED** [review]
See comments.

File: ccd-definition/AuthorisationCaseField/CareSupervision/eps-managing.json [hunk 8]
```diff
@@ -573,6 +573,34 @@
     "UserRole": "[EPSMANAGING]",
     "CRUD": "CRU"
   },
+  {
```
Comment (jamesrferguson1) [inline ccd-definition/AuthorisationCaseField/CareSupervision/eps-managing.json:576]: Is this missing sub header and placementRespondentsLabel?

File: ccd-definition/AuthorisationCaseField/CareSupervision/la-solicitor.json [hunk 9]
```diff
@@ -848,6 +848,20 @@
     "UserRole": "[LASOLICITOR]",
     "CRUD": "CRU"
   },
```
Comment (jamesrferguson1) [inline ccd-definition/AuthorisationCaseField/CareSupervision/la-solicitor.json:850]: Are the other 4 elements elsewhere in this file?

File: ccd-definition/AuthorisationCaseField/CareSupervision/superuser.json (hunk 3 above)
Comment (DanCatchpole) [reply ccd-definition/AuthorisationCaseField/CareSupervision/superuser.json:290]: Removed access to the ones given as the role doesn't have access to any of the other placement event fields

**Review by jamesrferguson1 — APPROVED** [review]
Adds ability to send placement notice to respondants

**Pull request merged**